sf_stats.Api/sf_stats.DataAccess.MSSql/EntityConfigurations/DivisionEntityConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+﻿using sf_stats.Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace sf_stats.DataAccess.MSSql.EntityConfigurations
+{
+    public class DivisionEntityConfiguration : IEntityTypeConfiguration<Division>
+    {
+        public void Configure(EntityTypeBuilder<Division> builder)
+        {
+            builder.HasKey(x => x.Id);
+            builder.Property(x => x.Id)
+                .UseIdentityColumn()
+                .ValueGeneratedOnAdd();
```

Comment:
I really should make this like a base entity class and have these entities inherit from it so we don't have to keep putting this in every new entity. More just thinking out loud here  don't think we need to change anything

Reply:
yeah, I like the idea, but then you have to inherit the base class and also call it from the `Configure` method, and I can't think of what else would be shared between everything

Reply:
I was thinking about adding a createdate and a lastmodifieddate to everything. How do you feel about that as like a base for every entity? @JoshuaTLand

Reply:
I like it

Reply:
But going to let you do it lol